### PR TITLE
Refs #120 and fixes (Sql Day1)

### PR DESCRIPTION
**Pull request by shekharx**

### CODE REVIEW

File: Modules/SQL/Day1/Shekhar chawla/day1.sql [hunk 1]
```diff
@@ -0,0 +1,175 @@
+-- Practice 1
+--Write a SQL statement to create a table named countries including columns CountryId, CountryName and RegionId and make sure that no countries except Italy, India and China will be entered in the table. and combination of columns CountryId and RegionId will be unique.
+
+CREATE TABLE countries(
```
Comment (radixdt12020) [inline Modules/SQL/Day1/Shekhar chawla/day1.sql:4]: C Should be capital

Comment (radixdt12020) [reply Modules/SQL/Day1/Shekhar chawla/day1.sql:4]: string value should be single

**Review by radixdt12020 — CHANGES_REQUESTED** [review]
make all these changes

**Review by radixdt12020 — CHANGES_REQUESTED** [review]
Convert All the Queries into MS Sql

**Review by MilitPanchasara — COMMENTED** [review]
Day3 => practice from videos not done. (like practice on COALESCE, EOMONTH, PARSE, TRY_PARSE & math functions)

**Review by MilitPanchasara — COMMENTED** [review]
Day4 => Video exercise not done. Practice on ROLLUP, CUBE etc

**Pull request merged**